File: Tutorial5-3aDagger-Android/src/main/java/com/example/tutorial5_1dagger_android/ToastMaker.java
package com.example.tutorial5_1dagger_android;

import android.content.Context;
import android.widget.Toast;

public class ToastMaker {

    private Context context;

    public ToastMaker(Context context) {
        this.context = context;
    }

    public void showToast(String message) {
        Toast.makeText(context, message, Toast.LENGTH_SHORT).show();
    }


}

File: Tutorial4-5SubComponentBuilders/src/main/java/com/example/tutorial4_5subcomponentbuilders/model/ToastMaker.kt
package com.example.tutorial4_5subcomponentbuilders.model

import android.content.Context
import android.widget.Toast

class ToastMaker(private val context: Context) {

    fun showToast(message: String) {
        Toast.makeText(context, message, Toast.LENGTH_SHORT).show()
    }

}